chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -351,6 +364,73 @@ integration-editBibliography-wrapper =
+integration-citationDialog-btn-settings =
+    .title = { general-open-settings }
+integration-citationDialog-btn-mode =
+    .title = {
+        $mode ->
+            [library] Switch to List Mode
+            [list] Switch to Library Mode
+            *[other] Switch Mode
+    }
+    .aria-label = {
+        $mode ->
+            [library] The dialog is in Library mode. Click to switch to List Mode.
+            [list] The dialog is in List mode. Click to switch to Library Mode.
+            *[other] Switch Mode
+    }
+integration-citationDialog-btn-accept = 
+    .title = { general-accept }
+integration-citationDialog-btn-cancel = 
+    .title = { general-cancel }
+
+integration-citationDialog-general-instructions = Use Left/Right Arrow to navigate the items of this citation. Press Tab to select items to add into this citation.
+Press { command-or-control } - { return-or-enter } to save edits to this citation. Press Escape to discard the changes and close the dialog.
+integration-citationDialog-enter-to-add-item = Press { return-or-enter } to add this item to the citation.
+
+integradion-citationDialog-search-for-items = Search for items to add to the citation
+integration-citationDialog-aria-bubble =
+    .aria-description = This item is included in the citation. Press space bar to customize the item. { integration-citationDialog-general-instructions }
+integration-citationDialog-single-input =
+    .placeholder = { integradion-citationDialog-search-for-items }
+    .aria-description = Press Tab to select items to add into this citation. Press Escape to discard the changes and close the dialog.
+integration-citationDialog-input =
+    .placeholder = { integradion-citationDialog-search-for-items }
+    .aria-description = { integration-citationDialog-general-instructions }
+integration-citationDialog-aria-item-list =
+    .aria-description = Use Up/Down Arrow to change item selection. { integration-citationDialog-enter-to-add-item }
+integration-citationDialog-aria-item-library =
+    .aria-description = Use Right/Left Arrow to change item selection. { integration-citationDialog-enter-to-add-item }
+integration-citationDialog-collections-table =
+    .aria-label = Collections.
+    .aria-description = Select a collection and press Tab to navigate its items.
+integration-citationDialog-items-table =
+    .aria-label = { integration-citationDialog-enter-to-add-item }
+integration-citationDialog-items-table-added =
+    .aria-label = This item has been added into the citation. Press { return-or-enter } to add it again or { backsace-or-delete } to remove it.
+integration-citationDialog-add-all = Add all
+integration-citationDialog-collapse-section = 
+    .title = Collapse section
+integration-citationDialog-bubble-empty = (no title)
+integration-citationDialog-duplicates-warning-title = Potential Duplicate Item
+integration-citationDialog-duplicates-warning-message = This citation already has an item with this title and creator. Add anyway?
```

Comment:
Let's actually move all these (and the existing ones in this file) to an integration.ftl. There are a ton of these, and (I think) they're all used in specific windows, so no reason to clutter this file.